charts/signoz/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: signoz
-version: 0.0.12
```

Comment:
@medicharlachiranjeevi can you please revert this change? we only bump up the chart version with bunch of changes together. Mostly post new signoz release.

Reply:
Sure